### ISSUE TITLE
x/sys/unix: SO_REUSEPORT undefined on linux/386 and linux/amd64

### ISSUE THREAD

**Issue by tummychow** [golang/go#16075]
Please answer these questions before submitting your issue. Thanks!
1. What version of Go are you using (`go version`)?

```
$ go version
go version go1.6.1 linux/amd64
```
1. What operating system and processor architecture are you using (`go env`)?

```
$ go env
GOARCH="amd64"
GOBIN=""
GOEXE=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="/home/sjung/go"
GORACE=""
GOROOT="/usr/local/go"
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GO15VENDOREXPERIMENT="1"
CC="gcc"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0"
CXX="g++"
CGO_ENABLED="1"
```
1. What did you do?
   If possible, provide a recipe for reproducing the error.
   A complete runnable program is good.
   A link on play.golang.org is best.

```
$ cat soreuseport.go
package main

import (
    "fmt"

    "golang.org/x/sys/unix"
)

func main() {
    fmt.Println(unix.SO_REUSEPORT)
}
$ go run soreuseport.go
```
1. What did you expect to see?

A number, on this architecture it would be 15 (0xf). For example, on darwin/amd64:

```
$ go run soreuseport.go
512
```
1. What did you see instead?

```
$ go run soreuseport.go
# command-line-arguments
./soreuseport.go:10: undefined: unix.SO_REUSEPORT
```

See also: https://github.com/golang/sys/search?q=SO_REUSEPORT+path%3Aunix

**Comment by tklauser** [comment]
Should be fixed by golang/sys@ea9bcade75cb9

**Comment by ianlancetaylor** [comment]
Works for me.  I think you just need to update your golang.org/x/sys repo.
